src/formatter-settings/assets/features/formatterSettings/utils/Highlight.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as React from "react";
+import * as hljs from "highlight.js";
+import "../../../../../../webview-resources/highlight.css";
+
+export function Highlighter(content: string, language?: string): JSX.Element {
```

Comment:
If we are sure it's just to preview the Java code, no need to pass the language as an argument. Instead, you should migrate this function from a util to a UI component.

Reply:
Sure.